CHANGELOG.md

```diff
@@ -35,10 +35,7 @@ see if you need to change any of these:
 - `PEcAn.logger::setLevel()` now invisibly returns the previously set logger level
 -  Warning messages for `model2netcdf.ed2()` coming from `ncdf4::ncvar_put()` now are prepended with the variable name for easier debugging (#3078)
 - Added optional `process_partial` argument to `model2netcdf.ED2()` to allow it to process existing output from failed runs.
-- Added litter_mass_content_of_water to standard_vars table
-- Added litter_mass_content_of_water to model2netcdf.SIPNET
-- Added all SIPNET state variables to read_restart and write_restart
-
+- Added Observation preparation functions into the SDA workflow, which supports AGB, LAI, Soil Carbon, and Soil moisture.
```

Comment:
When you add your messages to the changelog you shouldn't delete other people's records

Reply:
That was changed automatically.